web/client/components/data/query/enhancers/geometrydetails.js

```diff
@@ -0,0 +1,16 @@
+/**
+* Copyright 2018, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+const {compose, withHandlers} = require('recompose');
+const {debounce} = require("lodash");
+
+module.exports = compose(withHandlers((initProp) => {
```

Comment:
Can't we generalize this into a "debouncing" enhancer? I don't get the purpose of having single use enhancers in separate files